docs/docs/guides/artifact.mdx

```diff
@@ -45,73 +43,129 @@ nodes:
         optional: true
 ```
 
-We then have to load the modified schema definition into Infrahub.
+Load the modified schema into Infrahub:
 
 ```bash
 infrahubctl schema load /tmp/schema.yml
 ```
 
-## Creating a group and adding members
+## Step 2: Create a target group
 
-We need to create a group for every Artifact Definition that we will define. An artifact will be generated for every member that is part of that group.
+Create a Standard Group to define which objects will generate artifacts.
 
-Create a Standard Group with the name `DeviceGroup` and add `NetworkDevice` `switch1`, `switch2` and `switch3` as a member.
+1. Navigate to the Groups section in the web interface
+2. Create a new Standard Group named `DeviceGroup`
+3. Add your target devices (`switch1`, `switch2`, `switch3`) as members
 
-More information can be found in the [Creating a group guide](./groups.mdx).
+For detailed group creation steps, see [Creating a group](./groups.mdx).
 
 <center>
   <VideoPlayer url='https://www.youtube.com/watch?v=ASGMKZVLCbY' light />
 </center>
 
-## Creating an artifact definition
-
-In the last step we need to define an Artifact definition, which groups together a [transformation](../topics/transformation.mdx) with a target group and forms the definition of the artifact. Artifact definitions can be created via the Web Interface, via GraphQL or via a [Git repository](../topics/repository.mdx). In this guide we will be using the Git repository.
+## Step 3: Define the artifact generation
 
-Add the following contents to the end of the `.infrahub.yml` file at the root of the `device_config_render` repository.
+Add an artifact definition to your repository's `.infrahub.yml` file:
 
 ```yaml
 artifact_definitions:
-  - name: "device_configuration"
-    artifact_name: "Device configuration file"
+  - name: "Device configuration file"
+    artifact_name: "device_configuration"
     parameters:
       name: "name__value"
     content_type: "text/plain"
     targets: "DeviceGroup"
     transformation: "device_config_transform"
 ```
 
-This defines an artifact with the following properties:
+This configuration specifies:
 
-- **name**: a unique name for the artifact
-- **parameters**: the parameter to pass to the transformation GraphQL query, in this case this we will pass the name of the object (device) as the name parameter
-- **content type**: the content type for the resulting artifact
-- **targets**: the name of a group of which the members will be a target for this artifact
-- **transformation**: the Jinja2 or Python transformation that should be used
+- **name**: Human-readable description
```

Comment:
Name and artifact_name are inverted here
name must be unique and shouldn't have spaces